source/class/qx/html/Element.js

```diff
@@ -1678,17 +1677,13 @@ qx.Class.define("qx.html.Element", {
      * @param direct {Boolean?false} Whether the values should be applied
      *    directly (without queuing)
      * @return {qx.html.Element} this object (for chaining support)
-     */
-    setStyles(map, direct) {
+     */ setStyles(map, direct) {
```

Comment:
It looks like there are still formatting errors (caused by prettier?). This is one of many.

Reply:
Fixed them manually - hope i found all. Running prettier did not help

Reply:
Could you find some more?